XLS-0070d-credentials/README.md

```diff
@@ -579,24 +569,28 @@ It's much faster to have the credential ID included - it's easy to make sure tha
 
 ### A.7: Can a credential issuer delete their account?
 
-Yes, even if they still have issued credentials in existence.
+Yes, though the credentials they created will be deleted.
 
 ### A.8: Does a credential issuer have to have an on-chain account?
 
-No, though they must have a valid keypair. If the second method of creating a credential is used (via a signature from the issuer), the issuer can still attest to a credential without having to have an account on-chain. This is to ensure closer compliance with the W3C Verifiable Credentials spec (see section 9).
+Yes.
 
 ### A.9: How do I get a list of credentials that an issuer has issued?
 
-The answer to that question is still being investigated. One option would be a Clio API.
+The `account_objects` RPC will return a list of all `Credential` objects an account is involved in - both as `Subject` and `Issuer`. To get a list of just the ones an account has issued, just filter that list for `Credential`s that have that account as the `Issuer`.
+
+### A.10: How do I get a list of credentials that an account has been issued?
+
+The `account_objects` RPC will return a list of all `Credential` objects an account is involved in - both as `Subject` and `Issuer`. To get a list of just the ones an account has been issued, just filter that list for `Credential`s that have that account as the `Subject`.
 
-### A.10: Can I edit the list of credentials stored in `AuthorizeCredentials`?
+### A.11: Can I edit the list of credentials stored in `AuthorizeCredentials`?
 
 No, you have to delete the `DepositPreauth` object and recreate it with the new list.
```

Comment:
@mvadari  From an operational perspective, isn't this cumbersome? Between the deletion and re-creation of an updated `DepositPreauth` object, my account could receive un-warranted payments from un-welcome actors.

Is it not possible to update/modify the `DepositPreauth` ledger object on-the-fly ?

Reply:
With the way DepositPreauth works, this is a necessity. The hash of the object includes the set of credentials, so you can't update the list without deleting and recreating the object.